src/Resource.php

```diff
@@ -205,11 +205,11 @@ protected function mergeAttributes(array $attributes)
      *
      * @return Relationship[]
      */
-    public function getRelationships()
+    public function getRelationships($filter = true)
```

Comment:
Well, technically you're even breaking backwards compatibility with this change, as someone may have overwritten this class.

We can also add a new method `getUnfilteredRelationships` that simply returns the return value of `buildRelationships` (but as a public method).